stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -78,4 +90,18 @@ public class AttachmentGallery : ConstraintLayout {
             }
         )
     }
+
+    private fun configureImagesMenu() {
+        binding.menuButton.setOnClickListener {
+            context.getFragmentManager()?.let { fragmentManager ->
+                ImagesMenuDialogFragment.newInstance(imagesMenuTitle, imageList)
+                    .apply {
+                        setImageClickListener { position ->
+                            binding.attachmentGallery.setCurrentItem(position, true)
+                        }
```

Comment:
Feels to me like we should also dismiss the dialog after we select an image in it, because otherwise it blocks the image we've scrolled to. What do you think?

Reply:
Makes sense